### ISSUE TITLE
Barcode added to position

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#3395]
The following barcode was added to a position instead of the the cryovial immediately below it in the hierarchy. However, positions cannot be edited to remove the barcode.  If positions cannot be edited to remove barcodes, they should not allow barcodes to be added in the first place. 
Can I have this barcode deleted from its current position position, please? It needs to be added to the cryovial in that position. 
A8PY8

An error occurred while processing this page!
Message: ERROR: FAIL: Positions cannot be edited. Where: PL/pgSQL function updatecontainer(bigint,bigint,character varying,character varying,character varying,character varying,character varying,double precision,double precision,double precision,bigint,bigint,character varying,character varying,character varying) line 30 at RAISE
Detail: Parameter types passed (15): CF_SQL_BIGINT, CF_SQL_BIGINT, CF_SQL_VARCHAR, CF_SQL_VARCHAR, CF_SQL_VARCHAR, CF_SQL_VARCHAR, CF_SQL_VARCHAR, CF_SQL_DOUBLE, CF_SQL_DOUBLE, CF_SQL_DOUBLE, CF_SQL_BIGINT, CF_SQL_BIGINT, CF_SQL_VARCHAR, CF_SQL_VARCHAR, CF_SQL_VARCHAR

**Comment by dustymc** [comment]
Walk me through the steps that lead here, please.

**Comment by campmlc** [comment]
Me wearing wrong glasses and clicking the tiny little square next to
position in Find Container results display rather than the tiny little
square just under it next to the cryovial I actually intended to edit. Then
I added the barcode to the position using the scary red box. When I
realized the mistake, I couldn't undo it.

**Comment by dustymc** [comment]
>wrong glasses

Got my first pair of those not too long ago!

Was the position in a parent? Was there anything in the position? I think "scary red box" is the critical part of this (it might have to stay scary, I'm not sure yet) but the more I know the more holes I'm likely to find/plug.

**Comment by campmlc** [comment]
I went to https://arctos.database.museum/guid/MSB:Mamm:122112
Clicked on Part Location in Catalog Record
Saw something like this (but without the barcode A8PY8 next to position 14)
![image](https://user-images.githubusercontent.com/14808196/106223212-3a8d5500-619e-11eb-8cb9-f71e55ac3b5a.png)
Clicked the tiny box next to position 14 instead of the box below it for "NK 39381 M (cryovial)"
Clicked Edit Container
Scrolled down to scary red box at bottom
Added barcode A8PY8 which at the time was a cryovial label, because I need to use this tool to add barcodes to legacy DGR Locator cryovials that have labels but not barcodes.
If this hadn't been a position, and I'd added the barcode to the wrong container, I would delete the barcode from the record and save. This deletes the barcode.
Then I have to use Create Container to re-create the same barcode as a cryovial label, and go through the process again with the scary red box in the correct container. 
But since a position is uneditable (except for allowing me to add a barcode to it), I'm stuck.
